zanata-war/src/main/webapp/WEB-INF/layout/admin/contact_admin_modal.xhtml

```diff
@@ -7,78 +7,62 @@
   aria-labelledby="myModalLabel" aria-hidden="true">
 
   <script>
-    function cancelEmail() {
-      document.getElementById('#{rich:clientId('subject')}').value = '';
-      document.getElementById('#{rich:clientId('message')}').value = '';
+    $(document).ready(function() {
+      jQuery('#' + jqSelector('#{rich:clientId('contact-admin-subject')}')).
+        val('#{msgs['jsf.message.admin.inquiry.subject']}');
+    });
```

Comment:
What is the purpose of this? It looks like it is inserting a hard-coded string into a field that I can see below that should have the value `#{contactAdminAction.subject}`.

Is this setting the default value for it? Can we do that in contactAdminAction instead so that it is easier to figure out?